app/subscriber/src/features/home/Home.tsx

```diff
@@ -32,8 +32,7 @@ export const Home: React.FC = () => {
             contentTypes:
               filter.contentTypes.length > 0 ? filter.contentTypes : [ContentTypeName.PrintContent],
             startDate: filter.startDate ? filter.startDate : new Date().toDateString(),
-            // TODO: For testing purposes leaving this commented out
-            // status: ContentStatus.Published,
+            status: ContentStatus.Published,
```

Comment:
Also TNO-1262 (only show published content)

Reply:
This will 'technically' filter on published content, but it isn't the correct implementation.  Subscribers should by default only be searching the index that only contains published content.  This will ensure they receive the best performance as it will have far less content in it than the unpublished index.  You don't need to fix in this PR.  Don't close TNO-1262 until the `ContentService.FindWithElasticsearchAsync(...)` has an override to use the correct index.

Reply:
Sounds good, will create another PR for 1262